target/linux/lantiq/files-4.14/arch/mips/boot/dts/FRITZ7412.dts

```diff
@@ -0,0 +1,241 @@
+/dts-v1/;
```

Comment:
Please include licensing info for your DTS file, see
https://openwrt.org/submitting-patches#dts_checklist
e.g. a line like:
`// SPDX-License-Identifier: GPL-2.0-or-later OR MIT`
above the `/dts-v1/;`

Licensing with GPL 2.0+ or MIT would be very helpful for maintenance of this file.